### DISCUSSION TITLE
Collecting data

### DISCUSSION THREAD

**Discussion by gpabdo** [HomeSpan/HomeSpan#412] [Q&A]
Good afternoon,

I was wondering if someone could give me any pointers on collecting data from HomeSpan sensors so I can stash it in InfluxDB. I would like to find or create an agent that could query the devices every 10s and then forward that collected data (temp, air quality, valve status) to whatever. I have seen some very old libs like https://github.com/mrstegeman/hapclient but just wanted to make sure some data collector doesn't already exist before I try and modernize something so old. 

Thanks!

**Comment by FidelDworp** [comment]
I am also looking for this possibility.
Could you please keep me in the loop for this question?

Very grateful!

Groeten,
Filip Delannoy

Op 22 okt. 2022 om 01:58 heeft Greg ***@***.***> het volgende geschreven:
Good afternoon,

I was wondering if someone could give me any pointers on collecting data from HomeSpan sensors so I can stash it in InfluxDB. I would like to find or create an agent that could query the devices every 10s and then forward that collected data (temp, air quality, valve status) to whatever. I have seen some very old libs like https://github.com/mrstegeman/hapclient but just wanted to make sure some data collector doesn't already exist before I try and modernize something so old.

Thanks!

**Comment by blaisebarrette** [comment]
Have you looked at the WEBLOG() macro of Homespan? Maybe it could be used?
